content/docs/user-guide/basic-concepts/parameter.md

```diff
@@ -5,5 +5,5 @@ tooltip: >-
   Pipeline stages (defined in `dvc.yaml`) can depend on specific values inside
   an arbitrary YAML, JSON, TOML, or Python file (`params.yaml` by default).
   Stages are invalid (considered outdated) when any of their parameter values
-  change. See `dvc param`.
+  change. See [`dvc params`](/doc/command-reference/params).
```

Comment:
These are linked automatically:
```suggestion
  change. See `dvc params`.
```

Reply:
No they're not (see other basic-concepts content too). Maybe for separate issue to fix that.

Reply:
Oh this is in a concepts tooltip frontmatter... My bad 🤦 

I'll re-re-add it.